Bukkit/src/main/java/com/plotsquared/bukkit/generator/BukkitPlotGenerator.java

```diff
@@ -434,7 +436,10 @@ private final class BukkitPlotBiomeProvider extends BiomeProvider {
         private static final List<Biome> BIOMES;
 
         static {
-            ArrayList<Biome> biomes = new ArrayList<>(List.of(Biome.values()));
+
+            ArrayList<Biome> biomes = new ArrayList<>(Arrays.stream(Biome.values())
+                    .filter(b -> Registry.BIOME.get(b.getKey()) == null)
+                    .toList());
```

Comment:
`.collect(Collectors.toCollection(ArrayList::new))`, then the outer ArrayList creation can be dropped